libs/perun/components/src/lib/identities-page/identities-page.component.ts

```diff
@@ -129,12 +137,16 @@ export class IdentitiesPageComponent implements OnInit {
           location.reload();
         } else if (result === 'OK') {
           this.notificator.showSuccess(
-            this.translate.instant('IDENTITIES.SUCCESSFULLY_ADDED') as string
+            this.translate.instant(
```

Comment:
Use the perun translate service instead.

Reply:
The changes on the identities page were restored due to another final solution of the certificates page.